blocks/library/search/index.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Server-side rendering of the `core/search` block.
+ *
+ * @package gutenberg
```

Comment:
This file is going to be copied to WordPress so it should be `WordPress`. See another block for reference:
https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/categories/index.php#L5